roles/spark/README.md

```diff
@@ -53,7 +53,7 @@ Setup export of PCP metrics to a Spark worker thread.
   roles:
     - role: performancecopilot.metrics.spark
       vars:
-        spark_export_port: 44388
+    ### spark_export_port: 44388
```

Comment:
not sure what markdownlint was complaining about here, but these should not be commented out

Reply:
Yep, same again here - this is an example, and should not be commented like this.